native/cocos/core/Root.cpp

```diff
@@ -442,8 +472,10 @@ void Root::destroyLight(scene::Light *light) { // NOLINT(readability-convert-mem
     light->destroy();
 }
 
-scene::Camera *Root::createCamera() const {
-    return ccnew scene::Camera(_device);
+scene::Camera *Root::createCamera() {
+    auto *camera = ccnew scene::Camera(_device);
+    _allCameraList.emplace_back(camera);
```

Comment:
Where to erase camera from `Root::_allCameraList` if camera component is removed from Scene?

Reply:
Remove the variable of "_allCameraList ” after swapchain refactor, in next commit.